docs/ebnf_like.md

```diff
@@ -0,0 +1,14 @@
+# Syntax Diagrams
+
+A short summary of the elements used in syntax diagrams.
+
+| Syntax symbol     | Description                                               |
+| ----------------- | --------------------------------------------------------- |
+| **:=**            | production rule (*"non-terminal can be expanded to ..."*) |
+| **{** ... **}**   | 0..N occurrences (*"\*"*)                                 |

```

Comment:
All of the EBNF documents above allow for a trailing comma even on the last item. We should at least document, here, that although they are used that way, the expectation is that the final element in a list will not have a trailing comma.